packaging/install.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
-echo "###################################################################"
+echo "########################################################################"
 echo "Cloudify installer is ready!"
-echo "Edit /etc/cloudify/config.yaml, and run cfy_manager install"
-echo "(members of group wheel have write access)"
-echo "###################################################################"
+echo "Run cfy_manager install --private-ip <PRIV_IP> --public-ip <PUB_IP>"
+echo "Or edit /etc/cloudify/config.yaml to set the IPs and any other settings"
+echo "(members of group wheel have write access to the configuration file)"
+echo "########################################################################"
```

Comment:
Run: cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>
(Use cfy_manager -h for a full list of options)

Alternatively, edit /etc/cloudify/config.yaml to set IPs and other options,
and then run cfy_manager.

(Members of the 'wheel' group have write-level access to this configuration file)